src/utils/__tests__/spid.test.ts

```diff
@@ -0,0 +1,229 @@
+import { QueueClient } from "@azure/storage-queue";
+import { some } from "fp-ts/lib/Option";
+import { DOMParser } from "xmldom";
+import {
+  getFiscalNumberFromPayload,
+  getRequestIDFromRequest,
+  getRequestIDFromResponse,
+  makeSpidLogCallback
+} from "../spid";
+
+const aSAMLRequest = `<?xml version="1.0"?>
```

Comment:
for more tests readability, can you move this declaration inside a mock file like was done in `io-spid-commons`
[here](https://github.com/pagopa/io-spid-commons/blob/master/src/utils/__mocks__/saml.ts)

Reply:
fixed in https://github.com/pagopa/io-backend/pull/611/commits/d1f1f6cc945bdba3deacad2c3b4b2f83a8f71e44